src/lib/protocols/credentials/CredentialService.ts

```diff
@@ -103,7 +103,11 @@ export class CredentialService extends EventEmitter {
     credDef: CredDef
   ): Promise<CredentialRequestMessage> {
     const proverDid = connection.did;
-    const offer = MessageTransformer.toMessageInstance(credential.offer, CredentialOfferMessage);
+
+    // FIXME: credential.offer is already CredentialOfferMessage type
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    const offer = JsonTransformer.fromJSON(credential.offer, CredentialOfferMessage);
```

Comment:
I don't understand why credential.offer is already CredentialOfferMessage type but we still calling the transformation?